conbench/entities/commit.py

```diff
@@ -84,8 +98,8 @@ def repo_url(self) -> Optional[str]:
         Return a URL (string) or None. The returned string is guaranteed to
         start with 'http' and is guaranteed to not have a trailing slash.
 
-        The `None` case is here because I think the database may contain other
-        (non-URL) value types; and we need to phase those out.
+        The `None` case is here because I think that legacy databases may
```

Comment:
We can (and I think we should) create an `alembic` migration that doesn't change schema but instead just scans through each database to "correct" any rows displaying this old behavior. In that same PR, we could correct these handful of places that are nervous to handle the `repository` column.

Reply:
Yeah, I thought about 'fixing' old databases, but what's the disadvantage of not doing that? I like to save work here and there by not trying to polish 'old' data too much :-). It's always a valid question to wonder who even cares about current historical data in current Conbench deployments.